engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -86,6 +86,10 @@ object ProgramExecutionSupport {
     val callablesCallback: Consumer[ExpressionCall] = fun =>
       if (callStack.headOption.exists(_.expressionId == fun.getExpressionId)) {
         enterables += fun.getExpressionId -> fun.getCall
+      } else {
+        System.err.println(
```

Comment:
@4e6 I need your help. Please take the PR over and try to move it forward. 

```
[info] - should send method pointer updates of autoscope constructors


Ignoring update of 00000000-0000-00aa-8bae-ed8bb3344d4a with value FunctionCall[function=T.A[Enso_Test.Test.Main:2:5-5], arguments: []]
Ignoring update of 00000000-0000-00aa-8bae-ed8bb3344d4a with value FunctionCall[function=T.B[Enso_Test.Test.Main:3:5-7] x=_, arguments: [null]]
Ignoring update of 00000000-0000-00ac-a00c-7da891184fde with value FunctionCall[function=T.C[Enso_Test.Test.Main:4:5-9] y=_ z=_, arguments: [A, T.B[Enso_Test.Test.Main:3:5-7] x=_]]
```

I can see that the information about _autoscoped constructor_ being invoked gets here, but it is not processed by the `ProgramExecutionSupport`. Probably it arrives late, at unexpected time - that's expected: evaluation of_autoscoped constructors_ is delayed...

Reply:
Reverted in 3a986f04fb